content/tr/es6.md

```diff
@@ -1,23 +1,25 @@
 
-With io.js@1.x (V8 4.1+), all that complexity goes away. All harmony features are now logically split into three groups for **shipping**, **staged** and **in progress** features:
+io.js@1.x (V8 4.1+) sürümüyle beraber tüm bu karmaşıklık ortadan kalkmaktadır. Tüm ES6 (harmoni) özellikleri **taşınıyor**, **düzenleme** ve **geliştirme** olmak üzere mantıksal üç gruba ayrılmıştır. Bu özellikler;
 
-*   All **shipping** features, the ones that V8 has considered stable, like [generators](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/function*), [templates](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/template_strings), [new string methods](https://developer.mozilla.org/en-US/docs/Web/JavaScript/New_in_JavaScript/ECMAScript_6_support_in_Mozilla#Additions_to_the_String_object) and many others are turned **on by default on io.js** and do **NOT** require any kind of runtime flag.
-*   Then there are **staged** features which are almost-completed features that haven't been completely tested or updated to the latest spec yet and therefore are not considered stable by the V8 team (e.g. there might be some edge cases left to discover). This is probably the equivalent of the state of [generators](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/function*) on 3.26. These are the "use at your own risk" type of features that now require a runtime flag: `--es_staging` (or its synonym, `--harmony`).
-*   Finally, all **in progress** features can be activated individually by their respective harmony flag (e.g. `--harmony_arrow_functions`), although this is highly discouraged unless for testing purposes.
+*   Tüm **taşınıyor** özellikleri V8 tarafından kararlı olarak kabul görmüş, [generators](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/function*), [templates](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/template_strings), [yeni string metodları](https://developer.mozilla.org/en-US/docs/Web/JavaScript/New_in_JavaScript/ECMAScript_6_support_in_Mozilla#Additions_to_the_String_object) gibi özellikler ile **io.js tarafından varsayılan** olarak açılmış özelliklerdir ve herhangi bir çalışma bayrağına gerek **duymazlar**.
+*   **düzenleme** durumunda ki özellikler testleri tamamlanmamış veya son tanımlayıcı dökümanına uygun hale getirilmemiş ve V8 takımı tarafından kararlı olarak belirtilmemiş özellikleri içerir. (örn: there might be some edge cases left to discover). V8 3.26 sürümünde [generators](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/function*) bu duruma eşdeğer yapıdadır. Bu özellikleri kullanmakta ki riski göze alıyorsanız, `--es_staging` (veya eşanlamlısı `--harmony`) çalışma bayrağını kullanmanız gerekir.
```

Comment:
there might be some edge cases left to discover @ayhankuru bu satır kalmış, istersen kaldıralım. tam uygun birşey bulamadım. görüşün varsa değiştir istersen yoksa şimdilik kaldır güncelleriz.